### PR TITLE
CRIMRE-389 Surface appeal lodged date, change details and maat id in appeal cases

### PR DESCRIPTION
**Pull request by arthurashman**
## Description of change

In appeal cases surface appeal attributes:
- lodged date
- financial change details 
- previous maat id

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-389

### Before changes:
image

### After changes:
image

## How to manually test the feature
Put an appeal / appeal with changes case type through Apply, view Case Details section on Review

### CODE REVIEW

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 1]
```diff
@@ -7,20 +7,28 @@
     visit crime_application_path(application_id)
   end
 
+  it 'includes the page title' do
+    expect(page).to have_content I18n.t('crime_applications.show.page_title')
+  end
+
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:13]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 2]
```diff
@@ -7,20 +7,28 @@
...
   end
 
+  it 'shows that the application is unassigned' do
+    expect(page).to have_content('Assigned to: no one')
+  end
+
+  it 'includes button to assign' do
+    expect(page).to have_content('Assign to your list')
+  end
+
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:31]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 3]
```diff
@@ -41,95 +49,118 @@
...
-      super().deep_merge('case_details' => { 'offence_class' => 'C',
-                                              'offences' => [{ 'name' => 'Robbery',
-                                                              'offence_class' => 'C',
-                                                              'passportable' => true,
-                                                              'dates' => [{
-                                                                'date_from' => '2020-12-12',
-                                                                      'date_to' => nil
-                                                              }] }] })
-    end
+  context 'with offence class' do
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:126]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 4]
```diff
@@ -41,95 +49,118 @@
...
+    context 'when at least one of the offences is manually input' do
+      it 'does shows the class not determined badge' do
+        table_body = find(:xpath,
+                          "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
+                          //tr[contains(td[1], 'Non-listed offence, manually entered')]")
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:131]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 5]
```diff
@@ -165,6 +196,53 @@
...
+
+        expect(ioj_table).to have_content('More details about loss of liberty.')
+      end
+    end
+  end
+
+  it 'does not show the CTAs' do
+    expect(page).not_to have_content('Mark as completed')
+  end
+
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:245]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 6]
```diff
@@ -41,95 +49,118 @@
...
+        row = first(:xpath,
+                    "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
+                    //tr[contains(td[1], 'Robbery')]")
+
+        expect(row).to have_content('Class C')
+      end
+
+      it 'displays calculated overall offence class badgee' do
+        expect(page).to have_content('Overall offence class Class C')
+      end
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:163]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 7]
```diff
@@ -41,95 +49,118 @@
...
-      row = first(:xpath,
-                  "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
-                  //tr[contains(td[1], 'Robbery')]")
+        expect(table_body).to have_content('Class not determined')
+      end
 
-      expect(row).to have_content('Class C')
+      it 'displays undetermined overall offence class badge' do
+        expect(page).to have_content('Overall offence class Undetermined')
+      end
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:138]: Just reordered spec to match order of fields on page

File: spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb [hunk 8]
```diff
@@ -41,95 +49,118 @@
...
+        table_body = find(:xpath,
+                          "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
+                          //tr[contains(td[1], 'Non-listed offence, manually entered')]")
 
-    it 'does show the offence class' do
-      row = first(:xpath,
-                  "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
-                  //tr[contains(td[1], 'Robbery')]")
+        expect(table_body).to have_content('Class not determined')
+      end
```
Comment (arthurashman) [inline spec/system/viewing_an_application/that_is_open_and_unassigned_spec.rb:134]: Just reordered spec to match order of fields on page

**Review by willmcb — APPROVED** [review]
Looks good - perhaps we could think about having a presenter or something to avoid some view logic, but happy for this to be in a further PR etc.

**Pull request merged**